### ISSUE TITLE
X-Window placements when using external screen

### ISSUE THREAD

**Issue by pucicu** [XQuartz/XQuartz#344]
Sometimes, usually when XQuartz is running for long time and inbetween an external screen was connected and disconnected and again connected, new X-window apps pop up outside of any visible screen frame. I cannot see them directly, only when using macOS' expose functionality. Nevertheless, I cannot access them. Meanwhile, because it is so boring, I wrote a short script which collect all open X-windows  to a certain place in the screen. But then I have to reorder these windows, of course. This is particularly annoying when, e.g., a small question window with a confirmation dialog pops up. Then the entire app is blocked because I cannot access this confirmation dialog.

It works, when XQaurtz is started new after connecting an external screen.

After disconnecting the external screen it seems that X is forgetting some screen sizes of the external screen.

**Comment by jeremyhu** [comment]
I suspect this is related to https://gitlab.freedesktop.org/xorg/app/quartz-wm/-/issues/1

**Comment by pucicu** [comment]
Perhaps. But the mentioned issue was already closed 4 years ago and I still have this problem. Therefore, it is either another problem or some specific settings I'm not aware of which affect my window placement after reconnecing the external screen.

**Comment by jeremyhu** [comment]
Nope.  That issue has never been closed.

**Comment by pucicu** [comment]
you are right. it was not closed. but the posts gave me the impression that the issue was solved. sorry for not beeing correct in using the right terminology.
